triton-vm/src/table/extension_table.rs

```diff
@@ -419,7 +420,7 @@ pub trait Quotientable: ExtensionTable + Evaluable {
             let interpolant_degree = interpolant_degree(padded_height, num_trace_randomizers);
             let max_degrees = vec![interpolant_degree; self.full_width()];
             let zerofier_degree = 1;
-            self.dynamic_initial_constraints()
+            self.dynamic_initial_constraints(&AllChallenges::dummy())
```

Comment:
Why a `dummy()` here? It says that this is only used for debugging purpose, and that Fiat-Shamir is used in production. Is this not production?

Reply:
I suppose the name `dummy` is not a good fit here. Something along the lines of “placeholder” is better suited. The actual challenges cannot yet be computed in the methods where `AllChallenges::dummy()` is being called, but we still need to figure out the `max_degree`, and thus need placeholder challenges.

Reply:
Gotcha. The `dummy()` being used here specifically says it's not for production use. Maybe `placeholder()` + a comment that this is used before they can be calculated, or as dummy values in tests; sounds less like it's a mistake.

Reply:
Resolved in 7afd91914ab11b39b0fd306d6414f28c79f2039f